code/modules/virus2/predefined.dm

```diff
@@ -0,0 +1,128 @@
+//Pre-Defined Global Diseases for use by nanites/wendigo/xenomicrobes/ect.
+
+
+
+
+var/list/global_diseases = list()
+
+
+
+/proc/create_global_diseases()
+	for(var/datum/disease2/disease/predefined/disease_type in /datum/disease2/disease/predefined)
```

Comment:
this does not work at all, which makes me wonder how much testing these changes received

Reply:
I tested it multiple times including just now and it works fine. Drinking nanites/microbes/whatever gives whoever drank the reagent the global disease created by this proc

Reply:
I suppose I should be using subtypes of instead of this loop, though

Reply:
I really don't see how that's possible: this loop does nothing, global_disease is only supposed to be written to by this proc, and it clearly doesn't